help_function/src/onedpl_test_group_load.cpp

```diff
@@ -0,0 +1,223 @@
+// ====------ onedpl_test_group_load.cpp------------ *- C++ -* ----===//
+
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+// ===----------------------------------------------------------------------===//
+#include <sycl/sycl.hpp>
+#include <dpct/dpct.hpp>
+#include <dpct/dpl_utils.hpp>
+#include <iostream>
+#include <oneapi/dpl/iterator>
+
+template<dpct::group::load_algorithm T>
+bool test_load_blocked_striped() {

```

Comment:
Can we rename this to `test_group_load`?  It would make this more clear in my opinion.
Similarly below : `test_load_blocked_striped_standalone` -> `test_group_load_standalone`